_pages/dev/session-management.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Session Management
+feature:
+- name: Session Management
+  spa_version: 3.0
+  cx_version: 1905
+---
+
+{% capture version_note %}
+{{ site.version_note_part1 }} 3.0 {{ site.version_note_part2 }}
+{% endcapture %}
+
+{% include docs/feature_version.html content=version_note %}
+
+Spartacus 3.0 introduces a new way to manage user sessions, handle tokens, and perform authorizations. The following sections are intended to help you understand how session management works under the hood, how can you extend the underlying mechanism, and how you can interact with the auth module from outside of Spartacus.
+
+## Client Authentication and User Authentication
+
+From the beginning, Spartacus has included both client authentication and user authentication. Although this is not typical for web applications, it is necessary for Spartacus to work with the OCC API.
+
+There are a number of endpoints that work on behalf of logged-out users, such as registering, resetting a password, placing an order as a guest, and verifying an address. These endpoints require an access token to be sent with the request, and this access token needs to be retrieved by following the Client Credentials Flow that is defined by the OAuth specification. This is the client authentication.
+
+Meanwhile, user authentication is used for requests that are sent on behalf of specific user resources. For example, if you want to update your profile, you need to be logged in. When you log in, the server confirms your credentials and returns an application access token. This token is then used for all requests on your account, such as updating your profile, making cart modifications, and checking out.
+
+Prior to Spartacus 3.0, the code for both client authentication and user authentication was in the `AuthModule`, which included interceptors, services, and methods for facades all mixed together. In Spartacus 3.0, the `AuthModule` still contains client and user authentication, but this is now the result of importing two modules, the `UserAuthModule` and the `ClientAuthModule`. Each module is responsible for one type of authentication.
+
```

Comment:
In the original version, one of the sentences in the above paragraph said "Interceptors, services, methods in facade, everything was mixed together in this module."
I wasn't sure about "methods in facade", so I took a guess and modified it to "methods for facades".
Can you verify if this is correct? 🙂

Reply:
"methods for facades" sound strange. I would switch to something like "facade methods"

Reply:
👍